src/pools/cluster/fixed.ts

```diff
@@ -29,10 +30,9 @@ export interface ClusterPoolOptions extends PoolOptions<Worker> {
  * @author [Christopher Quadflieg](https://github.com/Shinigami92)
  * @since 2.0.0
  */
-export class FixedClusterPool<
-  Data = unknown,
-  Response = unknown
-> extends AbstractPool<Worker, Data, Response> {
+export class FixedClusterPool<Data = unknown, Response = unknown>
+  extends AbstractPool<Worker, Data, Response>
+  implements IPool<Worker, Data, Response> {
```

Comment:
We don't need the `implements IPool here` because it's already in `AbstractPool`

Reply:
A pool implements the interface pool interface, whatever it extends.